server/auth/auth_oauth.py

```diff
@@ -0,0 +1,176 @@
+from flask import session, request, redirect, current_app
+from server.auth.auth import AuthTypeClientBase, AuthTypeFactory
+from server.common.errors import AuthenticationError, ConfigurationError
+from urllib.parse import urlencode
+from urllib.request import urlopen
+import json
+
+# It is not required to have authlib or jose.
+# However, it is a configuration error to use this auth type if they are not installed.
+missingimport = []
```

Comment:
This is a global variable and therefore should be capitalized to MISSING_IMPORTS or something similar right?

https://www.python.org/dev/peps/pep-0008/#constants

Reply:
It a global, but not a constant.